src/helix/client/client_ext.rs

```diff
@@ -105,30 +123,101 @@ impl<'client, C: crate::HttpClient + Sync + 'client> HelixClient<'client, C> {
-        }
-        self.req_get(
-            helix::channels::GetChannelInformationRequest::broadcaster_ids(ids),
-            token,
-        )
-        .await
-        .map(|response| response.data)
+        futures::stream::iter(ids.chunks(100).collect::<Vec<_>>())
+            .map(move |c| {
+                let req = helix::channels::GetChannelInformationRequest::broadcaster_ids(c);
+                futures::stream::once(self.req_get(req, token)).boxed()
+            })
+            .flatten_unordered(None)
+            .map_ok(|resp| futures::stream::iter(resp.data.into_iter().map(Ok)))
+            .try_flatten_unordered(None)
+    }
+
+    /// Get multiple [Stream](helix::streams::Stream)s from user ids.
+    ///
+    /// # Examples
+    ///
+    /// ```rust, no_run
+    /// # #[tokio::main]
+    /// # async fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync + 'static>> {
+    /// # let client: helix::HelixClient<'static, twitch_api::client::DummyHttpClient> = helix::HelixClient::default();
+    /// # let token = twitch_oauth2::AccessToken::new("validtoken".to_string());
+    /// # let token = twitch_oauth2::UserToken::from_existing(&client, token, None, None).await?;
+    /// use twitch_api::{types, helix};
+    /// use futures::TryStreamExt;
+    ///
+    /// let live: Vec<helix::streams::Stream> = client.get_streams_from_ids(&["123456", "987654"][..].into(), &token).try_collect().await?;
+    /// # Ok(()) }
+    /// ```
+    pub fn get_streams_from_ids<T>(
+        &'client self,
+        ids: &'client types::Collection<'client, types::UserId>,
+        token: &'client T,
+    ) -> impl futures::Stream<Item = Result<helix::streams::Stream, ClientError<C>>>
+           + Send
+           + Unpin
+           + 'client
+    where
+        T: TwitchToken + Send + Sync + ?Sized,
+    {
+        let ids = ids.chunks(100).collect::<Vec<_>>();
+        futures::stream::iter(ids.into_iter().map(move |c| {
+            let req = helix::streams::GetStreamsRequest::user_ids(c).first(100);
+            make_stream(req, token, self, std::collections::VecDeque::from)
+        }))
+        .flatten_unordered(None)
+    }
+
+    /// Get multiple [Stream](helix::streams::Stream)s from user logins.
+    ///
+    /// # Examples
+    ///
+    /// ```rust, no_run
+    /// # #[tokio::main]
+    /// # async fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync + 'static>> {
+    /// # let client: helix::HelixClient<'static, twitch_api::client::DummyHttpClient> = helix::HelixClient::default();
+    /// # let token = twitch_oauth2::AccessToken::new("validtoken".to_string());
+    /// # let token = twitch_oauth2::UserToken::from_existing(&client, token, None, None).await?;
+    /// use twitch_api::{types, helix};
+    /// use futures::TryStreamExt;
+    ///
+    /// let live: Vec<helix::streams::Stream> = client.get_streams_from_logins(&["twitchdev", "justinfan"][..].into(), &token).try_collect().await?;
```

Comment:
```suggestion
    /// let live: Vec<helix::streams::Stream> = client
    ///     .get_streams_from_logins(&["twitchdev", "justinfan"][..].into(), &token)
    ///     .try_collect()
    ///     .await?;
```

Same here. Wrapping this into two lines will still have some overflow in the docs.